### ISSUE TITLE
Show busy icon when agent busy with action

### ISSUE THREAD

**Issue by matrx-software** [matrx-software/matrx#98]
<a href="https://github.com/thaije"><img src="https://avatars1.githubusercontent.com/u/2978372?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [thaije](https://github.com/thaije)**
_Wednesday Feb 26, 2020 at 15:58 GMT_
_Originally opened as https://github.com/matrxs/MATRXS/issues/97_

----

When the agent is busy with an action, show an hourglass or something similar to indicate the agent is busy.

**Comment by thaije** [comment]
A new optional property was added for (human)agents: `visualize_when_busy`. This boolean can be used to specify per (human)agent whether to show when the agent is busy with an action. However, the option can also be set globally in the [defaults.py ](https://github.com/matrx-software/matrx/blob/%2398_show_agent_busy_icon/matrx/defaults.py) file as `AGENTBODY_VIS_BUSY `. 

By default, a small gears loading icon is shown when the property is set to True, and the agent is doing something. 
![busy_black_small](https://user-images.githubusercontent.com/2978372/81186978-bbbf0f00-8fb3-11ea-83da-13b7a88157b1.gif)

I also made it in MATRX colours! 
![busy_matrx_small](https://user-images.githubusercontent.com/2978372/81187009-c5487700-8fb3-11ea-9614-81ab9c5369b6.gif)

It uses the already existing `is_blocked_by_action` property from agents to see whether agents are blocked or not. However, by default every action (including move actions) block 1 tick, so that you see a lot of flickering of the loading icons when moving an agent.
